### PR TITLE
Use xpath to do source/translation matching

### PR DESCRIPTION
**Pull request by n1k0sv**
@diegobz @matthewjackowski

### CODE REVIEW

File: txlive.phantomjs [hunk 1]
```diff
@@ -5,26 +5,29 @@ var address = args[1];
 
 page.onInitialized = function() {
   page.evaluate(function() {
-    if (!window.Transifex || !window.Transifex.live)
-      document.write('<script src="//cdn.transifex.com/live.js"></script>');
+    if (!window.Transifex || !window.Transifex.live) {
+      //document.write('<script src="//cdn.transifex.com/live.js"></script>');
+      document.write('<script src="//s3.amazonaws.com/tx-live-dev/live.js"></script>');
```
Comment (n1k0sv) [inline txlive.phantomjs]: IMPORTANT: This needs to be reverted as soon as live.js V42 (with xpath support) is deployed.

**Comment by diegobz** [comment]
@nbasili I've experienced the following code to get rid of the entries that have `source == translation`.

``` diff
diff --git a/export.js b/export.js
index b39e8fc..0827eed 100644
--- a/export.js
+++ b/export.js
@@ -34,7 +34,12 @@ function runOnTranslations(sourceContent, sourceXPaths, callback) {
     for (var xpath in translationXPaths) {
       var key = sourceXPaths[xpath];
       if (key) {
-        result[sourceContent[key]] = translationContent[translationXPaths[xpath]];
+        if (sourceContent[key] == translationContent[translationXPaths[xpath]]) {
+          console.log('Translation and source content are the same: ' + sourceContent[key]);
+        }
+        else {
+          result[sourceContent[key]] = translationContent[translationXPaths[xpath]];
+        }
       }
       else {
         console.log(
```

However I found that are lots of duplicates while the for loop is executed. Not sure whether it's live.js related though. It's not a big deal, but we might want to address it.

```
[diegobz@Rasther-2 txlive-import-tool (import-with-xpath)]$ node export.js http://www.vainglorygame.com/ http://jp.vainglorygame.com/ ja-home.json
Running live on http://www.vainglorygame.com/
Found 278 source strings and 492 xpaths
Running live on http://jp.vainglorygame.com/
Found 275 source strings and 490 xpaths
Translation and source content are the same:SA
Translation and source content are the same:▾
Translation and source content are the same:SA
Translation and source content are the same:OS
Translation and source content are the same:iOS
Translation and source content are the same:▾
Translation and source content are the same:OS
Translation and source content are the same:OS
Translation and source content are the same:iOS
Translation and source content are the same:▾
Translation and source content are the same:Subject
Translation and source content are the same:Submit
Translation and source content are the same:Vainglory
Translation and source content are the same:Community
Translation and source content are the same:Community
Translation and source content are the same:English
Translation and source content are the same:日本語
Translation and source content are the same:한국어
Translation and source content are the same:▾
Translation and source content are the same:English
Translation and source content are the same:English
Translation and source content are the same:日本語
Translation and source content are the same:한국어
Translation and source content are the same:Community
Translation and source content are the same:English
Translation and source content are the same:/
Translation and source content are the same:日本語
Translation and source content are the same:/
Translation and source content are the same:한국어
Translation and source content are the same:Vainglory Fire
Translation and source content are the same:Vainglory Twitch
Translation and source content are the same:Vainglory YouTube
Translation and source content are the same:Vainglory Facebook
Translation and source content are the same:Vainglory Reddit
Translation and source content are the same:Vainglory Twitter
Translation and source content are the same:Vainglory
Translation and source content are the same:Vainglory
Translation and source content are the same:©{{0}}2015 Super Evil Megacorp. All rights reserved. Apple and Apple logo are trademarks of Apple Inc., registered in the US and other countries. Apple is a service mark of Apple Inc. Android™ is a trademark of Google Inc. Google Play™ is a trademark of Google Inc.
Translation and source content are the same:|
Translation and source content are the same:Vainglory
Translation and source content are the same:‘Cloud Raider’ Vox Skin Reveal!
Translation and source content are the same:Vox's Tier I skin is coming in the next big update! Check out in-game video footage, lore, an artist interview and more.
Translation and source content are the same:Free Hero Rotation (Aug 18): Try Joule, Fortress, Taka & More!
Translation and source content are the same:Update 1.7 Notes: The ‘Player Love’ Patch
Translation and source content are the same:Free Hero Rotation (Aug. 25): Play Krul, Ardan, SAW & More
Translation and source content are the same:Free Hero Rotation (Aug. 25): Play Krul, Ardan, SAW & More
Translation and source content are the same:Aug 25, 2015
Translation and source content are the same:Tweet for Chance to Win Tickets to TwitchCon 2015!
Translation and source content are the same:Tweet for Chance to Win Tickets to TwitchCon 2015!
Translation and source content are the same:Aug 24, 2015
Translation and source content are the same:‘Cloud Raider’ Vox Skin Reveal!
Translation and source content are the same:‘Cloud Raider’ Vox Skin Reveal!
Translation and source content are the same:Aug 21, 2015
Translation and source content are the same:New ICE Box: Triple Tier 3!
Translation and source content are the same:New ICE Box: Triple Tier 3!
Translation and source content are the same:Aug 25, 2015
Translation and source content are the same:New ICE Box: The Roamer Box
Translation and source content are the same:New ICE Box: The Roamer Box
Translation and source content are the same:Aug 22, 2015
Translation and source content are the same:New ICE Box: The Laner Box
Translation and source content are the same:New ICE Box: The Laner Box
Translation and source content are the same:Aug 19, 2015
Translation and source content are the same:Free Hero Rotation (Aug 18): Try Fortress, Joule, Taka & More!
Translation and source content are the same:Free Hero Rotation (Aug 18): Try Fortress, Joule, Taka & More!
Translation and source content are the same:Aug 18, 2015
Translation and source content are the same:free
Translation and source content are the same:This hero is <strong>free</strong> to play.
Translation and source content are the same:free
Translation and source content are the same:This hero is <strong>free</strong> to play.
Translation and source content are the same:free
Translation and source content are the same:This hero is <strong>free</strong> to play.
Translation and source content are the same:free
Translation and source content are the same:This hero is <strong>free</strong> to play.
Translation and source content are the same:free
Translation and source content are the same:This hero is <strong>free</strong> to play.
Translation and source content are the same:Celeste lights the way for her team, calling upon her stars for both artillery and long-range reconnaissance. Though fragile, she is well-suited to fighting from the furthest edges of battle.
Translation and source content are the same:free
Translation and source content are the same:This hero is <strong>free</strong> to play.
Translation and source content are the same:Fortress
Translation and source content are the same:This Halcyon Well guardian is a snarling, frightening sight in the jungle. He stalks prey, closes in with teammates, flanks targets and creates challenging, multi-front confrontations. Stay near your allies and bite and claw your way to kills.
Translation and source content are the same:Fortress
Translation and source content are the same:Fortress
Translation and source content are the same:Rona
Translation and source content are the same:Rona is a berserker who tears through teams as easily as single opponents. She's all about getting into battle and crushing her enemies. Her short-range damage excels at at driving her enemies from her.
Translation and source content are the same:Rona
Translation and source content are the same:Rona
Translation and source content are the same:Vainglory | The MOBA Perfected For Touch
Translation and source content are the same:Vainglory | The MOBA Perfected For Touch
Writing to file ja-home.json
``
```

File: txlive.phantomjs (hunk 1 above)
Comment (matthewjackowski) [inline txlive.phantomjs]: We should have a better way to deploy changes like this.

**Comment by n1k0sv** [comment]
@diegobz Your diff seems legit. You have my green light to commit it.

**Comment by diegobz** [comment]
@nbasili Done.

**Pull request merged**